Solaris specific patch, not suitable for upstream.
This is needed as the setuptools_scm_git_archive module
is not available in Solaris.

--- cheroot-9.0.0/pyproject.toml
+++ cheroot-9.0.0/pyproject.toml
@@ -7,7 +7,6 @@ requires = [
 
     # Plugins
     "setuptools_scm[toml]>=3.5",
-    "setuptools_scm_git_archive>=1.1",
 ]
 build-backend = "setuptools.build_meta"
 
--- cheroot-9.0.0/setup.cfg
+++ cheroot-9.0.0/setup.cfg
@@ -55,7 +55,6 @@ packages = find:
 include_package_data = True
 setup_requires = 
 	setuptools_scm>=1.15.0
-	setuptools_scm_git_archive>=1.0
 install_requires = 
 	backports.functools_lru_cache; python_version < '3.3'
 	selectors2; python_version< '3.4'
